ministryofjustice/correspondence_tool_staff
# CT-2402 Restyle Case Status component as per new Prototype design

## Description
Alter Case Status section as per new prototype design. This change affects all Correspondence Types. Prototype design: https://cts-prototype.herokuapp.com/case-detail
 
## Self-review checklist

 
### Screenshots
![out](https://user-images.githubusercontent.com/47113046/64216485-fadf1280-ceb0-11e9-9bf8-cab52f74b0ec.gif)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2402
 
### Deployment
N/A
 
### Manual testing instructions
1. Log in as any user
2. Select a case and view its status
3. View the case status section. 
4. Resize the browser window to see the responsive layout
